backend/routes/web.php

```diff
@@ -17,3 +17,7 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
```

Comment:
```suggestion
// This route is unreachable in the deployed application (the front hands off the request to GraphQL).  
// This route is added to provide the the reverse routing to generate the URL in password reset emails.
```

Just wanted to add a comment for future us trying to figure out how this works. 😁

Reply:
Suggestion committed